adv_name.py

```diff
@@ -19,6 +19,7 @@
 import easyocr
 import time
 import torch
+import ipdb
```

Comment:
This comment is not related to this line but add the same changes as below to WebHandler/__init__.py too